Volume/Extensions/Url+Extension.swift

```diff
@@ -22,4 +34,15 @@ extension URL {
 
         return parameters
     }
+    
+    var isDeeplink: Bool {
+        guard let host = self.host
+        else { return false }
+        switch host {
```

Comment:
This wouldn't be scalable for multiple valid host cases. Although, I did make a change so that when we add more cases, we don't have to edit the `isDeeplink` body.